docs/sphinx/quickstart.rst

```diff
@@ -1,390 +0,0 @@
-..
```

Comment:
Hmm, did we decide to remove this completely or is this mistakenly removed?

Reply:
No, we did not decide something like that. I will fix it, probably it was deleted by mistake. Apologies